nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/test/DirectoryEntriesTest.java

```diff
@@ -105,6 +114,21 @@ public void testLocalDirectoryEntries() throws Exception {
         JSONAssert.assertEquals(continentLocalContentJson, result.getString(), JSONCompareMode.NON_EXTENSIBLE);
     }
 
+    @Test
+    public void shouldFailWhenDuplicatedEntriesExist() {
```

Comment:
This test presents several issues:
- it is not located in the right module, it has nothing to deal with automation. You can create a `TestDirectoryCSVLoader ` in `nuxeo-platform-directory-test` module.
- You make a cast to `SQLDirectory` which will only succeed on your a local/default test run but not on multidb jobs such as MongoDB. Also, cast looks like useless 🤔 
- Your call to `fail` should have a message

Reply:
Anyway, this test is not needed. `AbstractDirectoryTest#shouldFailWhenCreateDuplicateEntries` is enough to validate the error message for duplicated entries as both are using `session::createEntry`.

Reply:
regarding into the  different comment this tests will be removed. and AbstractDirectoryTest#shouldFailWhenCreateDuplicateEntries is enough to validate the needed behaviour.